### DISCUSSION TITLE
new maintainer

### DISCUSSION THREAD

**Discussion by sckott** [ropensci/charlatan#120] [General]
@kylevoyto Since you've contributed many times to the package, curious if you're interested in taking over maintainer role of the package? I can't maintain it anymore so looking for a replacement.

**Comment by kylevoyto** [comment]
@sckott Thank you for the consideration, but I do not do much R development anymore. I am not interested in taking over the maintainer role.

**Reply by sckott** [reply]
Okay, thanks for your response
